hikari/users.py

```diff
@@ -484,6 +489,11 @@ def banner_url(self) -> typing.Optional[files.URL]:
     @property
     def default_avatar_url(self) -> files.URL:
         """Default avatar URL for this user."""
+        if self.discriminator == "0":  # migrated account
+            return routes.CDN_DEFAULT_USER_AVATAR.compile_to_file(
+                urls.CDN_URL, discriminator=(self.id >> 22) % 6, file_format="png"
```

Comment:
```suggestion
                urls.CDN_URL, discriminator=self.id.created_at % 6, file_format="png"
```

Reply:
self.id.created_at is a datetime

also from my commit description:
> doing >> 22 instead of .created_at.timestamp() cuz that also adds DISCORD_EPOCH and I am too lazy to test if it changes it